packages/apps/graph/generated-schema.graphql

```diff
@@ -72,38 +78,55 @@ scalar DateTime
 """Floats that will have a value of 0 or more."""
 scalar Decimal
 
+"""A record of an execution of a function on a smart contract."""
 type Event implements Node {
   block: Block!
   chainId: BigInt!
+
+  """The block height of this event."""
   height: BigInt!
   id: ID!
   moduleName: String!
   name: String!
+
+  """
+  The order index of this event, in the case that there are multiple events.
+  """
   orderIndex: BigInt!
   parameterText: String!
+
+  """The module name and the event name combined."""
   qualifiedName: String!
   requestKey: String!
   transaction: Transaction
 }
 
-"""General information about the graph and chainweb-data"""
+"""General information about the graph and chainweb-data."""
 type GraphConfiguration {
+  """
+  The maximum confirmation depth configured in the environment variables.
+  """
   maximumConfirmationDepth: Int!
+
+  """The minimum block height that is available in the graph."""
   minimumBlockHeight: BigInt
 }
 
+"""Access rule for a certain smart contract or account."""
```

Comment:
```suggestion
"""Guard for an account."""
```
I would ignore the smart-contract part for now. We use it only on accounts here.